modules/search/class.jetpack-search.php

```diff
@@ -251,7 +251,6 @@ public function load_assets() {
 					'postTypeFilters'    => $widget_options['post_types'],
 					'postTypes'          => $post_type_labels,
 					'siteId'             => Jetpack::get_option( 'id' ),
-					'siteUrl'            => site_url(),
```

Comment:
Is this potentially used by the old (non-instant) Jetpack Search anywhere?

Reply:
It doesn't seem to be. 👍